### PR TITLE
Stricter server access

### PR DESCRIPTION
**Pull request by mjvkeulen**
## Description
Enforce use of a password in combination with an ssh-key server side.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -23,6 +17,8 @@ ifndef host
 host=staging
 endif
 
+ansible_play_default_flags = --ask-pass --ask-vault-pass
```
Comment (Larspolo) [inline Makefile:20]: why remove the file detection? Will be really useful when we separate staging from production key

Comment (mjvkeulen) [reply Makefile:20]: This check promotes keeping keys (including production-related keys) in plaintext, this should be discouraged.

Comment (Larspolo) [reply Makefile:20]: Then "become" should now also be removed

Comment (engelhamer) [inline Makefile:20]: > @Larspolo : become should be removed

Comment (Larspolo) [reply Makefile:20]: Also sshpass needs to get installed in `make setup`

Comment (mjvkeulen) [reply Makefile:20]: It is in the PR already

Comment (Larspolo) [reply Makefile:20]: Totally missed that, mb

**Pull request merged**